src/generic/editlbox.cpp

```diff
@@ -58,9 +58,9 @@ class CleverListCtrl : public wxListCtrl
     {
          int w = GetSize().x;
 #ifdef __WXMSW__
-         w -= wxSystemSettings::GetMetric(wxSYS_VSCROLL_X) + 6;
+         w -= wxSystemSettings::GetMetric(wxSYS_VSCROLL_X, this) + 6;
```

Comment:
Just a note (i.e. I don't say that anything needs to be changed here in this PR): we should use `FromDIP(6)` here.

Reply:
I'll have to test this. It might be just an offset between the scrollbar and column which can be the same for any DPI.